deeplake/core/tensor.py

```diff
@@ -1872,8 +1985,6 @@ def _verify_and_delete_vdb_indexes(self):
             raise Exception(f"An error occurred while deleting VDB indexes: {e}")
 
     def load_vdb_index(self, id: str):
-        if self.meta.htype != "embedding":
```

Comment:
we need to make sure that only tensors with expected htyps can be processed here please add check for that as it is done here https://github.com/activeloopai/deeplake/pull/2910/files#diff-ab9277d4753b6a0edbc487599564a90abb1ba63e62df825c5347029a618b7050R2026

Reply:
load_vdb_index is not applicable for inverted index as of now as direct search logic from python for inverted index is not there yet. Like what we have in hnsw i.e. searchknn. In case later if direct seach_inv or some other functions requirement comes then load_index will be utilized. Now search is only through tql.